### DISCUSSION TITLE
When will the new release be published?

### DISCUSSION THREAD

**Discussion by devmansurov** [lunarphp/lunar#140] [Q&A] [answered]
Already a lot of commits but no new releases. So I wanted to know when the new release will be?

**Comment by glennjacobs** [comment, marked as answer]
beta8 is out now.

This repo no longer contains the tagged releases, the releases are on the 'getcandy/admin' repo. 

See #138

**Comment by AzadGh95** [comment]
Hi @glennjacobs ,
When do you estimate the stable version will be released?
